mxcubeweb/core/components/samplechanger.py

```diff
@@ -135,6 +139,11 @@ def _addElement(parent, element):
 
             contents = {"name": root_name}
 
+            if hasattr(HWR.beamline.sample_changer, "get_room_temperature_mode"):
```

Comment:
It would probably be better to add the `get_room_tempereature_mode` on the `AbstractSampleChanger`  that by default returns `False`